cypress/e2e/footer.cy.ts

```diff
@@ -9,7 +9,7 @@ describe("Footer", () => {
     });
 
     it("renders the footer with links to contact, legal and imprint pages", () => {
-      cy.visit("/index.html");
+      cy.visitPortal("/index.html");
```

Comment:
@caebr Wir hatten bisher den built-in `visit` Command überschrieben, damit auf das Laden der Rollen gewartet wird. Leider habe ich keine gescheite Lösung gefunden den original Command unter einem anderen Namen verfügbar zu machen (damit ich den ohne Warten in den Tests verwenden kann). Deshalb gibt es halt nun das `visitPortal` (mit Warten), welches wir normalerweise verwenden sollten.